spec/factories/server.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :server do
+    sequence(:url) { |n| "https://test-#{n}.example.com/bigbluebutton/api/" }
+    sequence(:secret) { |n| "test-#{n}-secret" }
+    load { 1 }
+    online { true }
+    enabled { true }
+  end
+end
```

Comment:
While FactoryBot is generally used with ActiveRecord objects, the current implementation of the models and redis makes it work out of the box (for now).